inc/Ultilities/Page.class.php

```diff
@@ -150,57 +150,221 @@ public static function PageHome(){
         return $htmlHome;
     }
 
+    public static function PageMain(){
+        $htmlMain = '
+        <main>
+        </main>
+        ';
+        return $htmlMain;
+    }
+
     /**
      * store gallery function printer
      * @return string
      */
-    public static function PageStoreCatalog( $productList ) : string {
+    public static function PageStoreCatalog( $productList) : string {
 
         $htmlStoreCatalog = '<div class="store-gallery">';
 
         $htmlStoreCatalog .= self::storeFilter();
-        $htmlStoreCatalog .= self::storeCatalog($productList);
+        if(is_array($productList)){
+            $htmlStoreCatalog .= self::storeCatalog($productList);
+            
+        }else{
+            $htmlStoreCatalog .= self::storeEmpty();
+
+        }
             
         $htmlStoreCatalog .='</div>';
 
         return $htmlStoreCatalog;
     }
 
+    public static function storeEmpty(){
+
+        $htmlEmpty ='
+            <section class="emptyMsg">
+                <h2>SORRY ANY PRODUCT FOUND </h2>
+            </section>
+        ';
+
+        return $htmlEmpty;
+    }
+
     public static function storeFilter(){
```

Comment:
Make it private. We are only using inside Page. We do not need to export this one.